src/Model/ResourceField.php

```diff
@@ -53,7 +55,14 @@ class ResourceField extends DataObject
     public function getCMSFields()
     {
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            $fields->removeByName('Name');
+            $originalTitle = ReadonlyField::create('Name', i18n::_t(
+                __CLASS__ . '.OriginalTitle',
+                'Original title'
+            ))->setDescription(i18n::_t(
+                __CLASS__ . '.OriginalTitleDescription',
```

Comment:
Yeah I was wondering about that as I wrote them, ha. I should have checked, but I thought with the inconsistencies that it may be a rabbit hole, haha. But UPPER_SNAKE is fine, I'll update :)

Reply:
Why no auto-resolve :(
https://github.com/silverstripe/silverstripe-ckan-registry/compare/3f5b657cea14cd0b45351aeb17bdc3188e583e49..2e71603de96e7d8a22106ccd9ac39095f978bf45